terraform/services/ecs_service/alb.tf

```diff
@@ -8,8 +8,10 @@ resource "aws_alb_target_group" "ecs_service" {
   vpc_id   = "${var.vpc_id}"
 
   health_check {
+    port = 443
```

Comment:
oh, do remember to run `terraform fmt`

(Can you get IntelliJ to do this on save?)